### ISSUE TITLE
Solder Problem with upload?

### ISSUE THREAD

**Issue by Cybeez** [TechnicPack/TechnicSolder#423]

**Comment by spannerman79** [comment]
#442

**Comment by Cybeez** [comment]
Yeah but i dont understand how i can create the Zip with solder? I cant insert in the Modpack anywhere an Link for the Download because it is using Solder? Can you help me?

**Comment by spannerman79** [comment]
And as I stated [here](https://github.com/TechnicPack/TechnicSolder/issues/422#issuecomment-124304102)

> **This application is intended for experienced system administrators**

If you cannot read and understand http://docs.solder.io/ then don't use solder

**Comment by Cybeez** [comment]
Im here for help and not for any Other shit!

**Comment by spannerman79** [comment]
Then read http://docs.solder.io/

Once you've read through it at least once read through it again.  Your answer is there

**Comment by Cybeez** [comment]
I have Read! But i got an Error [B#308] 2015/07/28 13:16:32 [SCHWERWIEGEND] net.technicpack.rest.RestfulAPIException: Error accessing URL [http://------/solder/public/api/modpack/]

**Comment by Cybeez** [comment]
It does not insert the Modpack name? ?

**Comment by bochen415** [comment]
@Cybeez 
1. Watch your language
2. Check your api responses (for /api /api/modpack and /api/mod) as stated in http://docs.solder.io/docs/api if they are wrong fix them and return when they are ok.

**Comment by spannerman79** [comment]
First of all

*  If you did read the docs over at http://docs.solder.io/ you would have seen how to enable debug mode
*  You provided no logs, and you come here like a little entitled child jumping up and down throwing a tantrum with **no** respect for anyone.
*  If you did in fact set up solder **as stated in the docs**  for which so far I have given the link to **twice** you would not be getting that error.

I'm done helping you.

**Comment by Cybeez** [comment]
http://solder.technicpack.net/api/modpack/energistics-infiinity/ Dont know

**Comment by Cybeez** [comment]
But there http://149.202.144.198/solder/public/api/modpack/energistics-infiinity all is Works

**Comment by Cybeez** [comment]
Solder is Works, but not the Zipping from the Modpack?

**Comment by bochen415** [comment]
@Cybeez 
There's an edit button here designed to add new information to existing posts. Use it.
*Full* launcher logs and solder debug logs are a minimum to help you.

**Comment by Cybeez** [comment]
1. http://pastebin.com/q3sPJPPn -> Launcher logs
2. Solder logs arent :/

**Comment by bochen415** [comment]
Your web server is not configured properly.
Check apache configs from http://docs.solder.io/docs/example-web-server-configs
Take note of separate vhosts for solder and for the mods repository.

**Comment by Cybeez** [comment]
Yeah but the Problem is, i have only 1 IP so i cant use vhosts?

**Comment by rfox90** [comment]
You can use Vhosts with 1 IP. Just follow the configuration for your web server.
